librz/core/cbin.c

```diff
@@ -2833,7 +2833,7 @@ static RzList *section_flag_to_rzlist(RzBin *bin, ut64 flag) {
 
 static int bin_sections(RzCore *r, PJ *pj, int mode, ut64 laddr, int va, ut64 at, const char *name, const char *chksum, bool print_segments) {
 	RzBinSection *section;
-	RzBinInfo *info = NULL;
```

Comment:
It seems that `info` has never been got from file before, that is weird. Is there any reason to keep it `NULL`?